app/models/meeting.rb

```diff
@@ -21,7 +21,7 @@ class Meeting < ActiveRecord::Base
 
   has_many :members, -> { order 'name' }, through: :meeting_members,
                                           source: :user
-  has_many :meeting_members, foreign_key: :meetingid
+  has_many :meeting_members, foreign_key: :meetingid, dependent: :destroy
```

Comment:
I wonder if we could make this `dependent: :delete_all` ? We definitely want `dependent: :destroy` for the `Group has_many :meetings` association above so that meetings are instantiated and associated meeting_members get destroyed, but I can't think of a reason we would want to instantiate meeting_members before destroying them.